gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -54,6 +54,7 @@
 import java.util.Set;
 
 import perfetto.protos.PerfettoConfig;
+import perfetto.protos.PerfettoConfig.AndroidPowerConfig.BatteryCounters;
```

Comment:
If the lines below don't get too long, refer to them as:
`PerfettoConfig.AndroidPowerConfig.BatteryCounters.<foo>`

I've been avoiding importing inner classes of protos.